### DISCUSSION TITLE
求助，巨额报酬！！！急

### DISCUSSION THREAD

**Discussion by ikungei** [share121/inter-knot#2146] [其他]
登得上GitHub登不上绳网是什么情况怎么解决
报酬一摩拉

**Comment by HFOxin** [comment]
这可是脑力活，没有60原石说不过去

**Comment by yutong903** [comment]
这个委托我接了
先在浏览器里搜watt toolkit，下载完成后再勾选GitHub加速
再下载tampermonkey直接搜就行
再用这个网址www.绳网.top
进入后下载绳网小助手，点击蓝色字体可以直接跳转
下载完成后就可以使用绳网了<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Reply by ikungei** [reply]
我承认卡芙卡女士美貌盖世无双
感谢您的帮助
